### DISCUSSION TITLE
Questions about self-hosted renovate in gitlab CI pipelines

### DISCUSSION THREAD

**Discussion by john-mancini30** [renovatebot/renovate#11260] [Request Help] [answered]
I am running renovate CLI in a gitlab CI pipeline. I have a few questions about how things work. Since I self host gitlab, I had to copy 
https://gitlab.com/renovate-bot/renovate-runner into my own repo, and then I have an additional repo called `renovate` which is where the pipelines are executing, which runs this template. My additional args are `--autodiscover=true --autodiscover-filter=john.mancini/*`

1. Where exactly should `renovate.json` go? I have that file in the `renovate` repo, but any repos that it scans that don't have `renovate.json` give me `"msg":"Repository is disabled - skipping"`
2. I thought renovate is supposed to on board a new repo it finds, by committing a `renovate.json` file, but instead for me it just gives me the disabled log output from 1. 
3. I have a repo which does have a `renovate.json` file that looks like
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "baseBranches": ["master"],
  "packageRules": [
    {
      "matchPackagePrefixes": ["My.Package"],
      "ignoreUnstable": false,
      "rangeStrategy": "replace"
    }
  ]
}
```
Is there a way I can configure this to only make changes to nuget related dependencies?

**Comment by rarkins** [comment, marked as answer]
1. `renovate.json` is only for repo config and not to configure the bot itself with admin/global config
2. See viceice's answer below
3. You could include `"enabledManagers": ["nuget"]` at the top level

**Reply by viceice** [reply]
https://gitlab.com/renovate-bot/renovate-runner/-/blob/main/templates/_common.gitlab-ci.yml#L10

Onboarding is disabled for our gitlab templates for noise reduction.

Autodiscover is disabled because of security concerns.

https://docs.renovatebot.com/gitlab-bot-security/
